doc/cookbook/source/examples/gaussian_processes/sparse_gaussian_process_regression.rst

```diff
@@ -0,0 +1,49 @@
+==================================
+Sparse Gaussian Process Regression
```

Comment:
I think this notebook is only for Gaussian likelihood.
 
For other likelihoods (eg, t distribution and Logistic distribution), we should use Laplace method.